src/NuGet.Services.AzureSearch/Measure.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.AzureSearch
+{
+    public static class Measure
+    {
+        public static async Task<DurationMeasurement<T>> DurationWithValueAsync<T>(Func<Task<T>> getValueAsync)
```

Comment:
Maybe use the same argument name as below :"actAsync" ?

Reply:
Sounds good.